lib/questionnaires/model/item/answer/src/coding_answer_option_model.dart

```diff
@@ -132,7 +132,7 @@ class CodingAnswerOptionModel {
         ?.valueString;
 
     final optionPrefix = (plainOptionPrefix != null)
-        ? RenderingString.fromText(plainOptionPrefix)
+        ? RenderingString.fromText(plainOptionPrefix, extensions: extensions)
```

Comment:
I'm not sure if we should pass the `extensions` here or not. From what I see, these should affect the question option itself, but here we are setting them to `plainOptionPrefix`, which is by itself coming from one of the `extensions` too.

Reply:
This is also been addressed [here](https://github.com/sujrd/faiadashu/pull/49/commits/3598028e169d969b943d32e8b739ed21aebcca11). Thanks!